src/components/form/AddIndexerBlockchainNetworks/cmp.tsx

```diff
@@ -0,0 +1,126 @@
+import React from 'react'
+import {
+  Icon,
+  TextInput,
+  Button,
+  Dropdown,
+  DropdownOption,
+} from '@aleph-front/aleph-core'
+import {
+  useAddIndexerBlockchainNetworks,
+  useIndexerBlockchainNetworkItem,
+} from '@/hooks/form/useAddIndexerBlockchainNetworks'
+import {
+  IndexerBlockchainNetworkItemProps,
+  AddIndexerBlockchainNetworksProps,
+} from './types'
+import NoisyContainer from '@/components/common/NoisyContainer'
+
+const IndexerBlockchainNetworkItem = React.memo(
+  (props: IndexerBlockchainNetworkItemProps) => {
+    const { idCtrl, blockchainCtrl, rpcUrlCtrl, networks, handleRemove } =
+      useIndexerBlockchainNetworkItem(props)
+
+    return (
+      <>
+        <div tw="mt-4">
+          <TextInput
+            {...idCtrl.field}
+            {...idCtrl.fieldState}
+            label="Id"
+            placeholder="ethereum-mainnet"
+          />
+        </div>
+        <div tw="mt-4">
+          <Dropdown
+            {...blockchainCtrl.field}
+            {...blockchainCtrl.fieldState}
+            label="Blockchain"
+            placeholder="ethereum"
+          >
+            {networks.map((value) => (
+              <DropdownOption key={value} value={value}>
+                {value}
+              </DropdownOption>
+            ))}
+          </Dropdown>
+        </div>
+
+        <div tw="mt-4">
+          <TextInput
+            {...rpcUrlCtrl.field}
+            {...rpcUrlCtrl.fieldState}
+            label="RPC node url"
+            placeholder="https://eth.mainnet.rpc/url"
+          />
+        </div>
+        {/* <div tw="mt-4">
+          <TextInput
+            {...abiUrlCtrl.field}
+            {...abiUrlCtrl.fieldState}
+            label="ABI url"
+            placeholder="https://eth.scan/path-to-abi-address-will-be-replaced/$ADDRESS"
+          />
+        </div> */}
```

Comment:
Is this useful for debugging? Should we keep it?

Reply:
Yep, i let it on purpose, it is something that will come back sooner than later. So even that it is a bad dev practice. Will be faster to get it back later